src/Page.js

```diff
@@ -861,6 +863,21 @@ Page.prototype.getPluginConfig = function () {
   };
 };
 
+/**
+ * Collect page sources provided by plugins
```

Comment:
Seems a bit wrong to say "page sources", since puml files are not really pages? Maybe "file sources"?